release/models/network-instance/openconfig-network-instance-policy.yang

```diff
@@ -74,7 +74,7 @@ revision "2018-11-21" {
           on in the local network instance. The string
           must match one of /network-instances/
           network-instance/protocols/
-          protocol/identifier in the local network
+          protocol/name in the local network
```

Comment:
The typo correction is correct however it's worth pointing out that these constructs are heavily relying on description statements to describe the intent vs. formulating proper model restrictions to enforce existence of these list keys within a local NI context.

PR as-is LGTM but this modeling should be reworked as a subsequent issue as I'm noticing now.

It requires implementations to have to build special logic outside of the schema that can provide inconsistencies.  This identityref allows for _ANY_ protocol type to be referenced legally and any unbound string for the name.  Maybe the grouping should not be so loose especially when there is only 1 current usage

Reply:
Anything I can do to help it? I am implementer, and I needed this ambiguous point resolved. I would of course always welcome more clarity on the schema level, but for now correct description is better than nothing. I would prefer to keep an issue of matching against technically unbound space of potential protocol names separate.

Reply:
LGTM